src/SAML2/XML/md/Extensions.php

```diff
@@ -71,7 +71,10 @@ public static function getList(DOMElement $parent): array
                 && array_key_exists($node->namespaceURI, $supported)
                 && array_key_exists($node->localName, $supported[$node->namespaceURI])
             ) {
-                if ($node->namespaceURI === ALG::NS) {
+                if (
+                    $node->namespaceURI === ALG::NS
+                    || $node->namespaceURI === EntityAttributes::NS
```

Comment:
Bug?

Reply:
Nope, intentional..
Take a look at the file/method and you'll get it..
Instead of `new class(DOMElement)` we now have to call `class::fromXML()`, but only for the classes we've migrated.. We have to deal with a hybrid situation until we have migrated all of them..

Reply:
Ah, that makes full sense! Didn't notice what was the purpose, sorry 😅

Reply:
Does that mean this one's approved

Reply:
Didn't finish taking a look yet, sorry! Got distracted with other stuff...